webauthn-rs-core/src/core.rs

```diff
@@ -878,21 +938,15 @@ impl WebauthnCore {
         Ok(data.authenticator_data)
     }
 
-    /// Authenticate a set of credentials allowing the user verification policy to be set.
-    /// If no credentials are provided, this will start a discoverable credential authentication.
-    ///
-    /// Policy defines the require UserVerification policy. This MUST be consistent with the
-    /// credentials being used in the authentication.
-    ///
-    /// `allow_backup_eligible_upgrade` allows rejecting credentials whos backup eligibility
-    /// has changed between registration and authentication.
-    pub fn generate_challenge_authenticate(
+    /// Generate a new challenge builder for client authentication. This is the first
+    /// step in authentication of a credential. This function will return an
+    /// authentication builder allowing you to customise the parameters that will be
+    /// sent to the client.
+    pub fn new_challenge_authenticate_builder(
```

Comment:
Can this be `ChallengeAuthenticateBuilder::new()` instead? It doesn't look like it uses any fields in `WebauthnCore`, so it feels weird to put a constructor here.

Reply:
Yeah, but for consistency we already do the same with ChallengeRegisterBuilder.

Reply:
OK